### PR TITLE
HIVE-2232: Update AWS default instance configuration to match openshift-install values.

### PR DESCRIPTION
**Pull request by jstuever**
Update AWS default instance settings to match current openshift-install values.

https://issues.redhat.com/browse/HIVE-2232

### CODE REVIEW

**Comment by jstuever** [comment]
/retest
ci/prow/e2e failed due to client side throttling.

**Comment by 2uasimojo** [comment]
Since this only affects `hiveutil create-cluster`, I'm inclined to Just Merge It™ once CI is happy.

Latest e2e failure is on autoscaler. Since a) e2e does use `hiveutil create-cluster`, and b) you've (significantly) increased the moxy of each Machine being configured, is it possible [100 replicas](test/e2e/postinstall/machinesets/infra_test.go) is no longer sufficient to weigh them down enough to trigger the autoscaler?

**Comment by 2uasimojo** [comment]
> Latest e2e failure is on autoscaler. Since a) e2e does use `hiveutil create-cluster`, and b) you've (significantly) increased the moxy of each Machine being configured, is it possible [100 replicas](test/e2e/postinstall/machinesets/infra_test.go) is no longer sufficient to weigh them down enough to trigger the autoscaler?

This was a reasonable guess... but wrong :P

Looking at the MachinePool from any of the failed e2es ([example](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_hive/2042/pull-ci-openshift-hive-master-e2e/1666205197405261824/artifacts/e2e/test/artifacts/hive_machinepools.yaml)) we can see that the number of replicas is good (12), but something is wrong with the networks, and that's what's preventing all the corresponding Nodes from being created/healthy.

File: test/e2e/postinstall/machinesets/infra_test.go [hunk 1]
```diff
@@ -304,7 +304,7 @@ func TestAutoscalingMachinePool(t *testing.T) {
 			Name:      "busybox",
 		},
 		Spec: appsv1.DeploymentSpec{
-			Replicas: pointer.Int32(100),
+			Replicas: pointer.Int32(120),
```
Comment (2uasimojo) [inline test/e2e/postinstall/machinesets/infra_test.go:307]: we don't still need this delta, do we?

**Review by 2uasimojo — COMMENTED** [review]
Can we unwind the busybox bump? Also would you mind squashing?

File: test/e2e/postinstall/machinesets/infra_test.go (hunk 1 above)
Comment (jstuever) [reply test/e2e/postinstall/machinesets/infra_test.go]: [Done](https://github.com/openshift/hive/compare/197a73ccb4f9b9fdd60c60a241c4e597a85e47f4..58740ecacecf162721b337be3a1a67c11ea3578a)

**Comment by 2uasimojo** [comment]
Last three failz look the same as before: [1](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_hive/2042/pull-ci-openshift-hive-master-e2e/1668316872732315648/artifacts/e2e/test/artifacts/EXIT_hive_machinepools.yaml), [2](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_hive/2042/pull-ci-openshift-hive-master-e2e/1668355976643743744/artifacts/e2e/test/artifacts/EXIT_hive_machinepools.yaml), [3](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_hive/2042/pull-ci-openshift-hive-master-e2e/1668406516295667712/artifacts/e2e/test/artifacts/EXIT_hive_machinepools.yaml). I guess instance availability comes and goes in a given AZ?

**Comment by 2uasimojo** [comment]
Wohoo. 

/lgtm

**Pull request merged**